vendor/k8s.io/kubernetes/pkg/master/master.go

```diff
@@ -127,6 +127,8 @@ type Config struct {
 
 	EnableCoreControllers    bool
 	EndpointReconcilerConfig EndpointReconcilerConfig
+	ControllerStorage        controlleretcd.ControllerStorage
+	ReplicaSetStorage        replicasetetcd.ReplicaSetStorage
```

Comment:
I wasn't envisioning this.  I was envisioning a way to tweak the `StorageFactory` to say, "if you call `.New` with rc.legacy, use storageVersion=rc.v1 no matter what its cohabitors request